### ISSUE TITLE
Depth data

### ISSUE THREAD

**Issue by alicefasoli** [Junggy/HouseCat6D#4]
Hello, I'm interested in learning more about the depth images within the dataset. Specifically, I've observed directories labeled 'depth' across all splits and 'depth_gt' only within the test and validation sets. Do these directories contain real and synthetic depth data respectively?
If so, why is this data not available for the training set? Would it be possible to release them? It could greatly benefit my project.

**Comment by Junggy** [comment]
Hi! Thank you for your interest in our dataset!

Actually how you see is correct.
"depth_gt" is a rendered depth from meshes and camera pose and "depth" is a depth from the sensor.

We didnt include the rendered depth from the training set, however, its not hard to render

https://github.com/Junggy/HouseCat6D/blob/main/visualization/vis_obj.py#L107

depth_i in the vis_obj.py script is the rendered depth.
Assuming its possible to run the script from your side,
you could easily modify the script and save the depth from depth_i.
if you have issue with running the script, please let me know!

Hope this is useful!

**Comment by alicefasoli** [comment]
Thank you!

**Comment by alicefasoli** [comment]
Could you clarify how you save the rendered depths for the test and validation sets? It appears the pyrender ones are normalized compared to yours.

**Comment by Junggy** [comment]
Ah yes if you just save the depth as file, it will save it as 8bit image - not ideal for a depth map
I assume you will get  rendered depth value something 0.5~1 (means less than objects are around 0.5~1meter from the camera)
you should try saving it as

depth_16bit = (depth_i * 1000).astype(np.uint16) // this converts m to mm
cv2.imwrite(depth_name,depth_16bit)

and when you read depth

depth_16bit = cv2.imread(depth_name,-1).astype(np.uint16)/1000  // you read depth as mm, then convert to meter 

let me know if theres any issue on this!
